packages/ckeditor5-dev-tests/bin/check-dependencies.js

```diff
@@ -428,7 +428,7 @@ function findMisplacedDependencies( dependencies, devDependencies, dependenciesT
  * @returns {Boolean}
  */
 function isDevDependency( absolutePaths ) {
-	return !absolutePaths.some( absolutePath => absolutePath.match( /src|theme/ ) );
+	return !absolutePaths.some( absolutePath => absolutePath.match( /[/\\](src|theme)/ ) );
```

Comment:
I would just make sure we are checking the whole path segments. It does not cost anything but we will be a little safer:

```suggestion
	return !absolutePaths.some( absolutePath => absolutePath.match( /[/\\](src|theme)[/\\]/ ) );
```